scripts/guests.js

```diff
@@ -10,4 +10,21 @@
                 Add park area id as state
 
         Return HTML String
-*/
\ No newline at end of file
+*/
+
+import { getGuestsList } from "./database.js"
+
+
+export const guestsList = () => {
+    const guests = getGuestsList()
+    let htmlString = `<div class="guests">`
+
+    for (const guest of guests) {
+        htmlString += `<div class="guestList">
+        <div Name: ${guest.name} </div>
+        <div ${guest.areaId} </div>
+        </div>
+        `
+    }
+    htmlString += `</div>`
+}
```

Comment:
I believe you may have forgotten to add a return for the htmlString variable here